oasis.md

```diff
@@ -0,0 +1,67 @@
+External OASIS Repositories
+===========================
+
+We offer hosting of non-OSG CVMFS repositories on OASIS. This means that requests to create, rename, remove, or blanking OASIS repositories will come in as GOC tickets. This document contains instructions for handling those tickets.
+
+!!! note "Also see"
+    * [Policy for OSG Mirroring of External CVMFS repositories](https://opensciencegrid.github.io/technology/policy/external-oasis-repos/)
+    * [External OASIS repository](https://opensciencegrid.github.io/docs/data/external-oasis-repos/)
+
+Hosting a repository on OASIS
+-----------------------------
+
+1.  Ensure that the repository administrator is valid for the VO. This can be done by (a) OSG already having a
+    relationship with the person or (b) the contacting the VO manager to find out.  Review the URL and verify verify
+    that it is appropriate for the VO.
+
+1.  Add the repository URL in OIM under the VO's OASIS repository URLs. This should cause the repository's configuration
+    to be added to the GOC Stratum-0 within 15 minutes.
+
+1.  If the respository ends in a new domain name that has not been distributed before, place a copy of the
+    `domain.name.pub` public key from `domain.name` into `/srv/etc/keys` on both `oasis-replica` and
+    `oasis-replica-itb`. If you do not have that key, ask the repository service representative how to obtain it.  In
+    order to support CVMFS client versions 2.2.X, a symbolic link of `domain.name.conf` has to be made in
+    `/cvmfs/config-osg.opensciencegrid.org/etc/cvmfs/domain.d` pointing to `default.conf`. This symbolic link has to be
+    created on the `oasis-itb` machine's copy of the `config-osg.opensciencegrid.org` repository and then copied to
+    production with the `copy_config_osg` command on the oasis machine.
+
+1.  If the repository name matches `*.opensciencegrid.org` or `*.osgstorage.org`, respond to the ticket to ask the
+    administrator to continue with the next step; all other repositories (such as `*.egi.eu`) you are done.
+
+1.  Once the repository administrator has responded to the ticket, indicating that the BNL stratum 1 needs to be
```

Comment:
Let's not have the administrator indicate that the BNL stratum 1 needs to be updated -- let's just have them indicate that they are finished with their next step.